hax/hax/util.py

```diff
@@ -1393,6 +1451,12 @@ def update_process_status(self, event: ConfHaProcess) -> None:
         LOG.debug('Setting process status in KV: %s:%s', key, data)
         self.kv.kv_put(key, data)
 
+        if str(event.fid) in motr_processes_status:
```

Comment:
Please add a function to get and update a motr_process_status under a lock just to avoid any race conditions between multiple thread.

Reply:
Done